tools/generator/AnDOGenerator.py

```diff
@@ -123,14 +123,15 @@ def __init__(self, expName=None, guid=None, sesID=None, date=None, sesNumber=Non
     def get_session_path(self):
         path = os.path.join(f'exp-{self.expName}',
                             f'sub-{self.guid}',
-                            f'ses-{self.sesID}'
+                            f'ses-{self.sesID}',
+                            MANDATORY_SUBFOLDER
```

Comment:
you only need to add the folder below not here, since `get_all_folder_paths` is calling `get_session_path`. Now you would have two nested `ephy` subfolders.

Reply:
It doesn't seem to pass the test_paths function if I remove it :
![image](https://user-images.githubusercontent.com/17256769/105197479-f1edf000-5b3c-11eb-920a-ffff4d6e3746.png)
So maybe there is something that I don't understand ?

Reply:
Yes, but this is because you introduced 'ephys' also in the expected result. Before this PR it was just generating the session_path (so everything down to the session level, without the data folders).

Reply:
So should I delete the "ephys" in the expected result ?